src/net.h

```diff
@@ -557,6 +557,16 @@ class CNode
     // m_tx_relay == nullptr if we're not relaying transactions with this peer
     std::unique_ptr<TxRelay> m_tx_relay;
 
+    /** Whether we should relay transactions to this peer (their version
+     *  message did not include fRelay=false and this is not a block-relay-only
+     *  connection). This only changes from false to true. It will never change
```

Comment:
>This only changes from false to true

Could you mention that this happens only at the beginning of the connection lifetime?

Reply:
`m_relays_txs` can be set to true after initial connection, if the peer sends a `filterload` or `filterclear` message.

Reply:
Right, thanks.